doc/template/simple_form.md

```diff
@@ -46,6 +47,29 @@ Using the [Neat Grid](http://neat.bourbon.io/) to align label and corresponding
     = f.button :submit
 ```
 
+### Field Sizing
+
+As some fields do not require an entire line to themselves (e.g. Postcode), there are 3 different row styles to choose from:
+
+* 2-1
+* 1-2
+* 2-2
+
+Each number refers to the column size input fields can assume in a grid of 7 available columns. The remainder is used for labels. When choosing the sizing, label & input value length should be considered.
+
+**Usage:**
+
+* Extra `.row` and `.sized-[x]-[y]` class for the wrapper of the 2 fields
+
+```haml
+.row.sized-2-1
+  = form.input :suburb
+  = form.input :postcode
+.row.sized-2-2
```

Comment:
But does this fallback to the same issue have in Bootstrap about semantic classes?

People used to criticise me by using:

```haml
<div class="row">
  <div class="col-md-8">.col-md-8</div>
  <div class="col-md-4">.col-md-4</div>
</div>
```

in Timesheets.